### ISSUE TITLE
How to get resource stats?

### ISSUE THREAD

**Issue by Paxa** [mperham/connection_pool#94]
The only way I found:

```ruby
{
  max: pool.instance_variable_get(:@available).instance_variable_get(:@max),
  created: pool.instance_variable_get(:@available).instance_variable_get(:@created),
  busy: pool.instance_variable_get(:@available).instance_variable_get(:@que)
}
```

Is there more proper way to do it?

**Comment by mperham** [comment]
The stats will be available as soon as you send a PR to add them. 👍

**Comment by florrain** [comment]
+1

**Comment by mikebaldry** [comment]
So this, can be closed now, based on #104
